lib/PlaidClient.js

```diff
@@ -210,6 +211,37 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  function(access_token, start_date, end_date, cb) {
+    var transactions = [];
+    var self = this;
+
+    P.coroutine(function*() {
+      while (true) {
+        const transactionsResponse = yield self.getTransactions(
+          access_token,
+          start_date,
+          end_date,
+          {
+            count: 500, // largest allowed value
+            offset: transactions.length,
+          }
+        );
+
+        transactions = R.concat(
+          transactions, transactionsResponse.transactions);
+        if (transactions.length === transactionsResponse.total_transactions) {
```

Comment:
can we change this to be `>=` ? I am somewhat concerned about the possibility of this loop not terminating.

Reply:
makes sense, though something could be wrong with the plaid API in that case

Reply:
yes -- I think that it "should never happen"

better to be safe!